data/crac-io/crac-io-cim/src/main/java/com/powsybl/openrao/data/cracio/cim/craccreator/CimContingencyCreationContext.java

```diff
@@ -7,27 +7,19 @@
 
 package com.powsybl.openrao.data.cracio.cim.craccreator;
 
-import com.powsybl.openrao.data.cracio.commons.api.ElementaryCreationContext;
 import com.powsybl.openrao.data.cracio.commons.api.ImportStatus;
+import com.powsybl.openrao.data.cracio.commons.api.StandardElementaryCreationContext;
 
 /**
  * @author Godelaine de Montmorillon {@literal <godelaine.demontmorillon at rte-france.com>}
  */
-// TODO : promote getNativeName & getCreatedObjectId (but rename it getCreatedObjectId) to ElementaryCreationContext
-    // and remove this class, and those like it that do not need extra information
-public final class CimContingencyCreationContext implements ElementaryCreationContext {
-    private String contingencyID;
-    private String contingencyName;
-    private ImportStatus importStatus;
-    private String createdContingencyId;
-    private String importStatusDetail;
-    private boolean isAltered;
-
+// TODO :  remove this class, and those like it that do not need extra information
+public final class CimContingencyCreationContext extends StandardElementaryCreationContext {
```

Comment:
I think you can add "imported" and "notImported" static methods to StandardElementaryCreationContext and remove this class now